build/run-e2e-suite.sh

```diff
@@ -99,6 +99,7 @@ cd $reportsDir
 #for cmName in `k get cm -l junitreport=true -o json | jq  '.items[].binaryData | keys[]' | tr '\"' ' '`
 #do
 #
+#
```

Comment:
Unnecessary change

Reply:
Changed to trigger CI because changing only the workflow file does not trigger CI